esmvalcore/cmor/check.py

```diff
@@ -257,12 +257,28 @@ def _check_dim_names(self):
                 except iris.exceptions.CoordinateNotFoundError:
                     try:
                         coord = self._cube.coord(coordinate.standard_name)
-                        self.report_error(
-                            'Coordinate {0} has var name {1} instead of {2}',
-                            coordinate.name,
-                            coord.var_name,
-                            coordinate.out_name,
-                        )
+                        if self._cmor_var.table_type == 'CMIP6' and \
+                           coord.ndim > 1 and \
+                           coord.standard_name in ['latitude', 'longitude']:
```

Comment:
Are `latitude` and `longitude` the only possible coordinate names?
I remember there were other cases. @jvegasbsc ?

Reply:
Well, this is about `standard_name`s, and certainly `latitude` and `longitude` are the only correct ones here. According to pure CF 1.7 Sections [4.1](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/cf-conventions.html#latitude-coordinate) and [4.2](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/cf-conventions.html#longitude-coordinate) these standard names are optional, but CMIP 6 sharpens this to a strict requirement in the [model output requirements, p. 6, note 9](https://goo.gl/neswPr) in the present case of 2d coordinate variables.